xunit.runner.visualstudio.testadapter/VsTestRunner.cs

```diff
@@ -363,9 +369,7 @@ void RunTests(IRunContext runContext, IFrameworkHandle frameworkHandle, LoggerHe
             var assemblyFileName = runInfo.AssemblyFileName;
             var assemblyDisplayName = Path.GetFileNameWithoutExtension(assemblyFileName);
             var shadowCopy = assembly.Configuration.ShadowCopyOrDefault;
-            var appDomain = assembly.Configuration.AppDomain ?? AppDomainDefaultBehavior;
-            if (DisableAppDomainRequestedInRunContext(runContext.RunSettings.SettingsXml))
-                appDomain = AppDomainSupport.Denied;
+            var appDomain = !RunSettingsHelper.DisableAppDomain ? AppDomainSupport.Denied : (assembly.Configuration.AppDomain ?? AppDomainDefaultBehavior);
```

Comment:
Good catch--@drognanar, this is likely why you were seeing the strong name verification errors when running tests.